package.json

```diff
@@ -25,6 +25,7 @@
     "core-js": "^3.19.0",
     "csvjson-csv2json": "^5.0.6",
     "electron-store": "^8.0.1",
+    "file-saver": "^2.0.5",
```

Comment:
This doesn't seemed to be used anywhere, saving seems to use `fs`. I guess it can it be removed?